frontend/src/app/Settings/Facility/Components/FacilityInformation.tsx

```diff
@@ -26,110 +29,118 @@ const FacilityInformation: React.FC<Props> = ({
   };
 
   return (
-    <div className="usa-form usa-form--large">
-      <h2 className="font-heading-lg" style={{ margin: 0 }}>
-        Facility information
-      </h2>
-      <TextInput
-        label="Testing facility name"
-        name="name"
-        value={facility.name}
-        required
-        onChange={onChange}
-        onBlur={() => {
-          validateField("name");
-        }}
-        validationStatus={errors.name ? "error" : undefined}
-        errorMessage={errors.name}
+    <>
+      <Prompt
```

Comment:
I believe this prompt should be in the `FacilityForm` component not the `FacilityInformation` as it should be at the form level and `FacilityInformation` is a set of inputs that is reused across multiple forms

Reply:
done